pkg/client/factory.go

```diff
@@ -45,9 +45,9 @@ func (f OptimizelyFactory) Client(clientOptions ...OptionFunc) (*OptimizelyClien
 	appClient := &OptimizelyClient{
 		executionCtx:    executionCtx,
 		DecisionService: decision.NewCompositeService(f.SDKKey),
-		EventProcessor: event.NewEventProcessor(event.BatchSize(event.DefaultBatchSize),
-			event.QueueSize(event.DefaultEventQueueSize), event.FlushInterval(event.DefaultEventFlushInterval),
-			event.SDKKey(f.SDKKey)),
+		EventProcessor: event.NewEventProcessor(event.WithBatchSize(event.DefaultBatchSize),
```

Comment:
@mikeng13 shouldn't we call it `NewBatchEventProcessor`?

Reply:
Yeah that is correct. Let me make that change!